CHANGELOG.md

```diff
@@ -35,4 +36,25 @@ Additionally it includes many conformance test updates and fixes:
 - Increased default timeout
 - Removal of K8s.io/kubernetes dependency
 
+Please check out the [network-policy-api website](https://network-policy-api.sigs.k8s.io/) for more information.
+
+# v0.1.7
+
+API Version: v1alpha1
+
+This is a patch release of the network-policy-api. It includes two
+main resources geared towards cluster admins:
+
+- AdminNetworkPolicy
+- BaselineAdminNetworkPolicy
+
+The new aspects of the API being released here that are worth highlighting include:
+
+- A new type of egress peer `networks` is supported to be able to express CIDR ranges as peers
+- An experimental egress peer `nodes` is supported to be able to express Kubernetes nodes as peers
+- An experimental egress peer `domainNames` is supported to be able to express FQDNs as peers
+- Docs text change around calling out that host-networked pods are not selected as part of subject or peers
+- Removal of `sameLabels` and `notSameLabels`
```

Comment:
This should probably be a separate paragraph, with a bit more information about why, and future plans, rather than just hiding as a single bullet point...